src/FontCache.cc

```diff
@@ -337,13 +337,15 @@ void FontCache::check_cleanup()
       pos = it;
     }
   }
-  FT_Done_Face((*pos).second.first);
+  const FontFace *face = (*pos).second.first;
   this->cache.erase(pos);
+  FT_Done_Face(face->face_);
+  delete face;
```

Comment:
Maybe use smart pointers?

Reply:
I considered it, but did not want to do all the typing. In the end it was not a big change at all. Probably safer this way.